src/rsdiv/diversity/mmr.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional, Sequence
+
+import numpy as np
+import numpy.ma as ma
+
+from .base import BaseReranker
+
+
+class MaximalMarginalRelevance(BaseReranker):
+    def __init__(self, lbd: float):
+        self.lbd = lbd
```

Comment:
`lambda` should be limited to [0, 1], if not the result indices should be the same with `list(range(k))`.